crates/modelardb_compression/src/models/mod.rs

```diff
@@ -87,6 +87,14 @@ pub fn maximum_allowed_deviation(error_bound: ErrorBound, value: f64) -> f64 {
     }
 }
 
+/// Returns true if compression is lossless i.e., `error_bound` value is 0.
+pub fn is_lossless_compression(error_bound: ErrorBound) -> bool {
+    match error_bound {
+        ErrorBound::Absolute(error_bound) => error_bound == 0.0,
```

Comment:
Maybe create an issue to add an `ErrorBound::Lossless` enum variant instead of encoding lossless using `ErrorBound::Absolute()` and/or `ErrorBound::Relative(0)`. What do you think @CGodiksen and @chrthomsen?

Reply:
I am leaving this one as it is for now and will create an issue if other contributors also agree on that.

Reply:
@CGodiksen and @chrthomsen what are your opinion?